### PR TITLE
Get interface language from browser preferences

### PR DESCRIPTION
**Pull request by petzlux**

### CODE REVIEW

**Comment by petzlux** [comment]
review please

File: geoportailv3/static/js/maincontroller.js [hunk 1]
```diff
@@ -140,7 +143,14 @@ app.MainController = function($scope, gettextCatalog, appExclusionManager,
 
   this.setMap_();
 
-  this.switchLanguage('fr');
+  /**
+   * @type {string}
+   * @private
+   */
+  this.browserLanguage_ =
+      ngeoGetBrowserLanguage(goog.object.getKeys(this.langUrls_));
```
Comment (elemoine) [inline geoportailv3/static/js/maincontroller.js]: Do you need that private property?

Comment (elemoine) [inline geoportailv3/static/js/maincontroller.js]: I mean why not do the following:

```js
  var browserLanguage = ngeoGetBrowserLanguage(goog.object.getKeys(this.langUrls_));
  this.switchLanguage(browserLanguage);
```

**Comment by elemoine** [comment]
I just added a comment, otherwise looks good to me.

**Comment by elemoine** [comment]
Looks good to me. Please merge when Travis is happy.

**Pull request merged**